src/uu/stty/src/stty.rs

```diff
@@ -102,7 +103,11 @@ impl<'a> Options<'a> {
             all: matches.get_flag(options::ALL),
             save: matches.get_flag(options::SAVE),
             file: match matches.get_one::<String>(options::FILE) {
-                Some(_f) => todo!(),
+                Some(f) => std::fs::OpenOptions::new()
+                    .read(true)
+                    .custom_flags(libc::O_NONBLOCK)
```

Comment:
Could you add a comment explaining why NONBLOCK is necessary?

Reply:
Yes this is a good idea. It is basically "because that is what GNU stty does" and they say:

    Set the line opened by the file name specified in DEVICE instead of
    the tty line connected to standard input.  This option is necessary
    because opening a POSIX tty requires use of the ‘O_NONDELAY’ flag
    to prevent a POSIX tty from blocking until the carrier detect line
    is high if the ‘clocal’ flag is not set.  Hence, it is not always
    possible to allow the shell to open the device in the traditional
    manner.

(Note: They say `O_NONDELAY` but *mean* `O_NONBLOCK`. There is no such thing as `O_NONDELAY`.)

Reply:
Interesting! That definitely makes sense to mention in the code.